Dockerfile

```diff
@@ -13,6 +13,11 @@ RUN php -r "copy('https://getcomposer.org/installer', '/tmp/composer-setup.php')
 RUN php /tmp/composer-setup.php --install-dir=/usr/local/bin --filename=composer
 RUN composer global require silverstripe/vendor-plugin-helper
 
+# NVM
+ENV NVM_DIR=/root/.nvm
+RUN curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.2/install.sh | bash
```

Comment:
Personally a bit eh about piping a random Gist into bash

Reply:
That's a reasonable concern - but there aren't any good alternatives to this approach*, and it's no more dangerous than cloning the NVM repo and running the script from that, as both are trusting the developer of NVM to have appropriate security practices in place. Might need some feedback from @madmatt.

* - I _guess_ we could say "we trust this exact copy" and host it ourselves somewhere..

Reply:
@madmatt Any thoughts here?

Reply:
How about a checksum match on the file before executing it? I take it the file will never change because it's a tagged version.